kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/manager/ResourceManager.java

```diff
@@ -158,6 +182,38 @@ public final <T extends HasMetadata> void deleteResource(T... resources) {
         }
     }
 
+    public void deleteResources() {
+        if (!storedResources.containsKey(getTestContext().getDisplayName()) || storedResources.get(getTestContext().getDisplayName()).isEmpty()) {
+            LOGGER.info("In context {} is everything deleted", getTestContext().getDisplayName());
+        }
+        else {
+            LOGGER.info("Deleting all resources for {}", getTestContext().getDisplayName());
+        }
+
+        // if stack is created for specific test suite or test case
+        AtomicInteger numberOfResources = storedResources.get(getTestContext().getDisplayName()) != null
```

Comment:
I don't see why you need to count the number of resources.  Surely, you can just delete until the stack is empty.